examples/factors/rust-toolchain

```diff
@@ -0,0 +1,4 @@
+[toolchain]
```

Comment:
I think this file need to live wherever the workspace lives

Reply:
It seems to be happy with this file removed altogether (presumably it's using the top-level toolchain file)

Reply:
I think it needs to be alongside the workspace. One way to test is to blow away your `target` dir and perhaps tell `rustup` to remove the target triple also.

Reply:
Gotcha. Blowing away the `target` dir didn't cause problems, but I didn't have `rustup` remove the target triple.